src/components/PickupBox/PickupBox.jsx

```diff
@@ -0,0 +1,172 @@
+import {
+  Accordion,
+  AccordionItem,
+  AccordionPanel,
+  AccordionButton,
+  AccordionIcon,
+  ChakraProvider,
+  FormControl,
+  FormLabel,
+  Input,
+  Textarea,
+} from '@chakra-ui/react';
+
+import PropTypes from 'prop-types';
+import styles from './PickupBox.module.css';
+import RejectBoxPopup from '../AlertPopups/RejectBoxPopup/RejectBoxPopup';
+import ApproveBoxIcon from '../BoxIcons/ApproveBoxIcon.svg';
+import RejectBoxIcon from '../BoxIcons/RejectBoxIcon.svg';
+import PickupBoxIcon from '../BoxIcons/PickupBoxIcon.svg';
+import ImageVector from '../BoxIcons/ImageVector.svg';
+import { FYABackend } from '../../common/utils';
+
+const PickupBox = ({
+  approved,
+  boxID,
+  boxHolderName,
+  boxHolderEmail,
+  zipCode,
+  picture,
+  date,
+  status,
+  rejectionReason,
+  fetchBoxes,
+  pickup,
+}) => {
+  // A state for determining whether or not the rejectBoxPopup is open
+  // This state is set true when the reject button is clicked
+  const [rejectBoxPopupIsOpen, setRejectBoxPopupIsOpen] = useState(false);
+
+  // A function that updates the approved boolean in the backend and refreshes all boxes that are under review
+  // This method is called when the approve box icon is clicked
+  const approvePickupBoxFromUR = async id => {
+    FYABackend.put('/boxHistory/update', {
+      boxID: id,
+    }).then(async () => {
+      await fetchBoxes('under review', true);
+    });
+  };
+
+  return (
+    <ChakraProvider>
+      <div
+        className={`box
+        ${status === 'evaluated' && approved === true ? styles['box-approved'] : ''}
+        ${status === 'evaluated' && approved === false ? styles['box-rejected'] : ''}`}
+      >
+        <Accordion allowToggle>
+          <AccordionItem>
+            {/* Pickup box ID and date */}
+            <h3>
+              <AccordionButton>
+                <img src={PickupBoxIcon} alt="" />
+                <div className={styles['title-div']}>
+                  <p className={styles.title}>
+                    <p className={styles['box-number']}>Box #{boxID}</p>
+                    {date}
+                  </p>
+                </div>
+                <div className={styles['arrow-button']}>
+                  <AccordionIcon />
+                </div>
+              </AccordionButton>
+            </h3>
+            {/* Box details */}
+            <AccordionPanel className={styles['accordion-panel']} pb={4}>
+              <div className={styles['box-details']}>
+                {picture !== null && (
+                  <img src={picture} alt="" className={styles['pickup-image-corners']} />
+                )}
+                {picture === null && (
+                  <div className={styles['image-box']}>
+                    <img className={styles['image-vector']} src={ImageVector} alt="" />
+                  </div>
+                )}
+                <FormControl>
+                  {/* Box name */}
+                  <FormLabel htmlFor="name" marginTop="5%">
```

Comment:
nit: Can we move the margin-top to the css file for all of these and use the same classname? This will make it a lot easier if you want to change that value later on (you'll just need to change it in one place rather than 4-5 times).